### ISSUE TITLE
Past Hackathons page

### ISSUE THREAD

**Issue by aedwardg** [CodecademyCommunity/CodecademyCommunity.github.io#7]
## Purpose
One page I'd like to have on this site is a page where we showcase past hackathons.  We could include a description of the hackathon, the top teams, the discord handles of the winners and/or the winning project's github repo.

### Is your feature request related to a problem? Please describe.
We need to have more incentive for the community to engage in our hackathons! Hoping this might help.

## Proposed Solution
Open to any/all attempts or suggestions!

## Acceptance Criteria
Please list the requirements the implementation of this feature should meet.
- [ ] Page should include features discussed above, and any others you think would be cool.
- [ ] If you can wireframe, a screenshot of that would be helpful!

**Comment by aedwardg** [comment]
Will need to circle back with @fedGL to see if this is going to continue being a thing before we make a page for it.

**Comment by lyallstewart** [comment]
Yeah makes sense, if it is going to we'll also need to gather together some information for the past two hackathons and get things like the winning team's repo/discord handles together (if of course they are OK with that infomation being published - we would have to ask them)

**Comment by lyallstewart** [comment]
So thanks for confirming Fede, as hackathons are going to continue to be a thing I think the page can go ahead (although I'll leave the "needs-discussion" label on for now until we work out exactly what information we want to have displayed about the hackthons/teams) 😄

**Comment by aedwardg** [comment]
Based on our recent staff meeting, it seems like there's a fair chance Hackathons won't be continuing on the server, so I'm marking this as on-hold.
